addons/area_markers/functions/fnc_isInEditMode.sqf

```diff
@@ -0,0 +1,21 @@
+#include "script_component.hpp"
+/*
+ * Author: mharis001
+ * Checks if the Zeus display is in marker editing mode.
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * In Marker Editing Mode <BOOL>
+ *
+ * Example:
+ * [] call zen_area_markers_fnc_isInEditMode
+ *
+ * Public: No
+ */
+
+visibleMap
+&& {!dialog}
+&& {!call EFUNC(common,isInScreenshotMode)}
+&& {RscDisplayCurator_sections select 0 == 3}
```

Comment:
We should probably define the different tabs with preprocessors

Reply:
Will fix in rest of the code base as well.